Revit_oM/Requests/ViewTemplateByNameRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Data.Requests;
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.Revit
+{
+    [Description("IRequest that filters all view templates in document.")]
+    public class ViewTemplateByNameRequest : IRequest
+    {
+        /***************************************************/
+        /****                Properties                 ****/
+        /***************************************************/
+
+        [Description("Revit view template name. If blank all View Templates are returned instead.")]
```

Comment:
`If blank...` - this should be left for the method, it does not serve as helpful on this description - unless you're intending this to be used on default create methods?
Also, `Revit view template name` - see previous comments about how I feel on this :wink: